circle-test.sh

```diff
@@ -67,7 +67,7 @@ case $CIRCLE_NODE_INDEX in
         rc=${PIPESTATUS[0]}
         ;;
     1)
-        go test $PARALLELISM $TIMEOUT -v -race ./... 2>&1 | tee $CIRCLE_ARTIFACTS/test_logs_race.txt
```

Comment:
Jason and I are running pprof to dig into this now.  We are creeping up to 13gb of memory when doing a parallel race test.  Anything above 4gb will fail on circle.